miso-web/src/it/java/uk/ac/bbsrc/tgac/miso/webapp/integrationtest/ServiceRecordPageIT.java

```diff
@@ -171,7 +173,8 @@ public void testRemoveValues() throws Exception {
     assertServiceRecordAttributes(fields, savedRecord);
   }
 
-  private static void assertServiceRecordAttributes(Map<Field, String> expectedValues, ServiceRecord sr) {
+  private static void assertServiceRecordAttributes(Map<Field, String> expectedValues, ServiceRecord sr)
+      throws IOException {
     assertAttribute(Field.ID, expectedValues, Long.toString(sr.getId()));
     // TODO: assertAttribute(Field.INSTRUMENT, expectedValues,
     // instrumentService.getInstrument(sr).getName());
```

Comment:
The instrument should still be displayed on the page, and should still be validated - this should not be commented out or removed